services/brand-service.js

```diff
@@ -0,0 +1,49 @@
+const brandRepo = require('../repositories/brand-repository');
+const categoryRepo = require('../repositories/category-repository');
+
+const { constants, logger } = global;
+
+exports.getBrandsByCategoryId = (categoryId, callback) => {
+  const promise = categoryRepo.getBrandByCategoryId(categoryId);
+
+  promise.then((value) => {
+    const brandsGetFromRepo = value.brands;
+    const data = brandsGetFromRepo.map(brand => ({
```

Comment:
Test thử đoạn này chưa ông? Tại lần trước t viết thế này không ra đúng kết quả nên toàn phải viết:
const data = brands.map((brand) => {
 return {
     id: brand.id,
     brandName: brand.name,
 };
});
Ông dùng postman test lại nhá rồi reply lại comment này. @NguyenSonArsenal